README.md

```diff
@@ -11,8 +11,30 @@ organisation of neuroimaging data.
 In this repository, we develop the
 [BIDS specification](https://bids-specification.readthedocs.io/en/latest/).
 
-**Want to learn more about working with BIDS? Have a question, comment, or suggestion?**
-Open or comment on one of our [NeuroStars issues](https://neurostars.org/tags/bids) or check out the [BIDS Starter Kit](https://github.com/bids-standard/bids-starter-kit)!
+# Formatting your data with BIDS
+
+To organize your data in BIDS, all you need is neuro data, a computer, and the
+[BIDS specification](https://bids-specification.readthedocs.io/en/stable/).
+
+BIDS currently supports MRI, MEG, EEG, iEEG, behavioral, and phsyiological data
+and more modalities are being added continuously (see here for
+[work in progress modalities](https://bids-specification.readthedocs.io/en/stable/06-extensions.html#bids-extension-proposals)).
+
+As a dataset curator, the GitHub repository and all files therein can be safely ignored
+to focus on the rendered content instead (see link above).
+The specification is provided in the form of a webpage, built using
+[MkDocs](https://www.mkdocs.org/) and [Read the Docs](https://readthedocs.org/).
+
+# Getting started
+
+*Want to learn more about working with BIDS? Have a question, comment, or suggestion?*
+
+1. Read some introductory material, most likely the very basic problems have already been addressed!
+  - [BIDS Starter Kit](https://github.com/bids-standard/bids-starter-kit) for tutorials, wikis, templates, ...
+2. Post your question in one of several channels where BIDS members are active
+  - the [NeuroStars](https://neurostars.org/tags/bids) discourse forum, for technical questions
```

Comment:
perhaps removing the detail of ` for technical questions` - we can use this for all questions